pkg/serverless/tags/tags.go

```diff
@@ -17,31 +17,48 @@ import (
 )
 
 const (
+	// Environment variables for Unified Service Tagging
+	envEnvVar     = "DD_ENV"
+	versionEnvVar = "DD_VERSION"
+	serviceEnvVar = "DD_SERVICE"
+
+	// Environment variables for the Lambda execution environment info
 	qualifierEnvVar = "AWS_LAMBDA_FUNCTION_VERSION"
-	envEnvVar       = "DD_ENV"
-	versionEnvVar   = "DD_VERSION"
-	serviceEnvVar   = "DD_SERVICE"
 	runtimeVar      = "AWS_EXECUTION_ENV"
 	memorySizeVar   = "AWS_LAMBDA_FUNCTION_MEMORY_SIZE"
 
+	// FunctionARNKey is the tag key for a function's arn
+	FunctionARNKey = "function_arn"
+	// FunctionNameKey is the tag key for a function's name
+	FunctionNameKey = "functionname"
+	// ExecutedVersionKey is the tag key for a function's executed version
+	ExecutedVersionKey = "executedversion"
+	// RuntimeKey is the tag key for a function's runtime (e.g node, python)
+	RuntimeKey = "runtime"
+	// MemorySizeKey is the tag key for a function's allocated memory size
+	MemorySizeKey = "memorysize"
+	// ArchitectureKey is the tag key for a function's architecture (e.g. amd64, arm64)
```

Comment:
Nit: I think AWS uses the tag x86_64 instead of amd64, might want to double check that though